root/static/scripts/common/entity.js

```diff
@@ -231,8 +231,14 @@ import formatTrackLength from './utility/formatTrackLength';
 
     class Label extends CoreEntity {
         selectionMessage() {
+            let code = this.toJSON().label_code;
+            if (code) {
+                code = ` (Label Code: LC-${"0".repeat(5 - String(code).length)}${code})`;
```

Comment:
We can use the existing `formatLabelCode` for this. You'll need to import it:

`import formatLabelCode from '../../../utility/formatLabelCode';`

Also, this should be translatable. So, something like:

```suggestion
                codeSection = ' ' + bracketed(exp.l(
                    'Label Code: {code}',
                    {code: formatLabelCode(code)},
                ));
```

We use lowercase "Label code" on the label sidebar, FWIW. Not sure about that one.

Reply:
Added the helper function and 'bracketed' import. Fixed to lower case "Label code"